### ISSUE TITLE
Can't get version of kraken, kraken-report or mafft ?

### ISSUE THREAD

**Issue by tseemann** [sanger-pathogens/Roary#312]
Roary 3.7.0 and 3.8.0 installed via CPAN.

```
2017/02/28 14:48:36 Looking for 'kraken' - found /home/linuxbrew/.linuxbrew/bin/kraken
Use of uninitialized value in concatenation (.) or string at /bio/perl5/lib/perl5/Bio/Roary/External/CheckTools.pm line 129.
2017/02/28 14:48:36 Determined kraken version is
2017/02/28 14:48:36 Looking for 'kraken-report' - found /home/linuxbrew/.linuxbrew/bin/kraken-report
Use of uninitialized value in concatenation (.) or string at /bio/perl5/lib/perl5/Bio/Roary/External/CheckTools.pm line 129.
2017/02/28 14:48:36 Determined kraken-report version is
2017/02/28 14:48:36 Looking for 'mafft' - found /home/linuxbrew/.linuxbrew/bin/mafft
Use of uninitialized value in concatenation (.) or string at /bio/perl5/lib/perl5/Bio/Roary/External/CheckTools.pm line 129.       2017/02/28 14:48:36 Determined mafft version is
```

**Comment by andrewjpage** [comment]
Fixed getting the version for MAFFT

**Comment by tseemann** [comment]
```
$ kraken --version
Kraken version 0.10.5-beta
Copyright 2013-2015, Derrick Wood (dwood@cs.jhu.edu)

$ kraken-report --version
Kraken version 0.10.5-beta
Copyright 2013-2015, Derrick Wood (dwood@cs.jhu.edu)
```

**Comment by andrewjpage** [comment]
Fixed in version 3.10.2
